lib/src/serialization/tracking_state.dart

```diff
@@ -0,0 +1,21 @@
+import '../../data_types/tracking_state.dart';
+
+TrackingStateChange deserializeTrackingState(String event) {
```

Comment:
are these all tracking states? interesting

Reply:
to be changed